### PR TITLE
feat(testing): support cypress v13

### PR DESCRIPTION
**Pull request by barbados-clemens**
## Current Behavior

Nx does not supports cypress v13

## Expected Behavior

Nx supports v13 w/migration

## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by jordanpowell88 — APPROVED** [review]
LGTM FWIW

File: packages/cypress/plugins/cypress-preset.ts [hunk 1]
```diff
@@ -7,7 +7,6 @@ import vitePreprocessor from '../src/plugins/preprocessor-vite';
 interface BaseCypressPreset {
   videosFolder: string;
   screenshotsFolder: string;
-  video: boolean;
```
Comment (ndcunningham) [inline packages/cypress/plugins/cypress-preset.ts:10]: would this be breaking if you used to use the `video` property?

Comment (barbados-clemens) [reply packages/cypress/plugins/cypress-preset.ts:10]: Talked with Jack and we are standardizing on what cypress is doing which is defaulting video off for v13. people can easily turn it back on in their respective cypress.config.ts.

**Review by ndcunningham — APPROVED** [review]

**Review by isaacplmann — APPROVED** [review]

File: packages/cypress/src/migrations/update-16-8-0/cypress-13.spec.ts [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+  let tree: Tree;
+  beforeEach(() => {
+    tree = createTreeWithEmptyWorkspace({ layout: 'apps-libs' });
+  });
+
+  it('should update deps to cypress v13', async () => {
+    setup(tree, { name: 'my-app' });
+
+    await updateToCypress13(tree);
+    console.log(readJson(tree, 'package.json'));
```
Comment (leosvelperez) [inline packages/cypress/src/migrations/update-16-8-0/cypress-13.spec.ts:15]:
```suggestion
```

File: packages/cypress/src/migrations/update-16-8-0/cypress-13.ts [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    json.devDependencies ??= {};
+    json.devDependencies.cypress = '^13.0.0';
+    return json;
+  });
+
+  await formatFiles(tree);
+
+  return () => {
+    installPackagesTask(tree);
+  };
```
Comment (leosvelperez) [inline packages/cypress/src/migrations/update-16-8-0/cypress-13.ts:62]: You can remove this. The `migrate` command ignores it. It doesn't execute any migration generator callbacks. Instead, it checks whether the `package.json` has changed after all migrations have run, and if so, it performs an install.

**Review by leosvelperez — APPROVED** [review]
Just left some non-blocking comments.

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**